### DISCUSSION TITLE
Pages with the same name

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#6646] [General]
Hi

I've got a few pages with the same name. 

<img width="932" alt="Screenshot 2022-09-06 at 20 48 09" src="https://user-images.githubusercontent.com/72254741/188725562-61fa7c64-a56c-48bc-8fc4-69dc0ab7209e.png">

I've noticed when adding entries to the navigation builder there's no way of telling which `about` page is which?

<img width="1173" alt="Screenshot 2022-09-06 at 19 14 37" src="https://user-images.githubusercontent.com/72254741/188725577-fa049c71-58ea-43c4-bc7d-fded41b6fdc0.png">

Other than renaming the pages, is there a workaround for this?

```
Antlers: runtime
Version: 3.3.34 PRO
```

**Comment by edalzell** [comment]
Ya we run into that all the time too, would be a great feature request.

What would make sense there, the url?

**Reply by stephenmeehanuk** [reply]
In this example, it'd be handy to show the path to the page

```
about
friction-management/about
automation/about
```

This could work when it's a structured collection. 

But I'm not sure what could be used to distinguish same name entries if the collection wasn't structured.

**Comment by stephenmeehanuk** [comment]
Where's the link for feature requests?

**Reply by edalzell** [reply]
https://github.com/statamic/ideas/issues/new/choose

**Reply by stephenmeehanuk** [reply]
Done, thanks
https://github.com/statamic/ideas/issues/858
